src/main/java/org/mozilla/zest/core/v1/ZestOutputWriter.java

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.zest.core.v1;
+
+/**
+ * @since 0.14
+ */
+public interface ZestOutputWriter {
```

Comment:
Don't think we need to introduce this class just because of that debug statement (I was just kidding with the lost debug statement), but it might be useful in other places? Maybe `ZestRuntime` should extend it (that's the interface that already provides these methods)? Instead of `ZestBasicRunner` implement it.